src/components/learner-credit-management/cards/tests/CourseCard.test.jsx

```diff
@@ -20,13 +21,20 @@ import {
   useBudgetId,
   useSubsidyAccessPolicy,
   useEnterpriseFlexGroups,
+  useCatalogContainsContentItemsMultipleQueries,
 } from '../../data';
 import { getButtonElement, queryClient } from '../../../test/testUtils';
 
 import EnterpriseAccessApiService from '../../../../data/services/EnterpriseAccessApiService';
 import { BudgetDetailPageContext } from '../../BudgetDetailPageWrapper';
 import { EMAIL_ADDRESSES_INPUT_VALUE_DEBOUNCE_DELAY } from '../data';
 import { getGroupMemberEmails } from '../../data/hooks/useEnterpriseFlexGroups';
+import { ENTERPRISE_RESTRICTION_TYPE } from '../../data/constants';
+
+jest.mock('@edx/frontend-platform', () => ({
+  ensureConfig: jest.fn(),
+  getConfig: jest.fn(),
```

Comment:
nit: Might recommend defaulting to returning an empty object vs. undefined, should a test case in this test suite _not_ mock `getConfig` explicitly. That way, accessing properties on `getConfig` in implementation won't throw a JS error if an explicit mock happens to be forgotten.

Something like:

```jsx
jest.fn(() => ({}))
```